models/object_tracking_vittrack/README.md

```diff
@@ -5,10 +5,26 @@ VIT tracker(vision transformer tracker) is a much better model for  real-time ob
 video demo: https://youtu.be/MJiPnu1ZQRI
  In target tracking tasks, the score is an important indicator that can  indicate whether the current target is lost. In the video, vit tracker  can track the target and display the current score in the upper left  corner of the video. When the target is lost, the score drops  significantly. While nanotrack will only return 0.9 score in any  situation, so that we cannot determine whether the target is lost.
 
+This model is contributed by [Pengyu Liu](https://github.com/lpylpy0514) in GSoC 2023 project [**Realtime object tracking models**](https://github.com/opencv/opencv/wiki/GSoC_2023#idea-realtime-object-tracking-models)
+
 **NOTE: OpenCV > 4.8.0**
 
+# Demo
+
+```python
+#  tracking on video
+python demo.py --input /path/to/video
+
+# get help regarding various parameters
+python demo.py --help
+```
+
+# Example outputs
+
+![](README.assets/vittrack_demo.gif)
```

Comment:
directory name "README.assets" -> "example_outputs". We use this directory name for other models as well.